integration/src/test/java/info/rmapproject/indexing/kafka/SaveOffsetOnRebalanceIT.java

```diff
@@ -1,9 +1,10 @@
 package info.rmapproject.indexing.kafka;
 
 import static info.rmapproject.indexing.IndexUtils.EventDirection.TARGET;
+import static info.rmapproject.indexing.TestUtils.createSystemAgent;
```

Comment:
Same comment here on imports as on the previous IT.  Generally, "optimize imports" produces quite a bit of noise in a diff, and makes it hard to see what exactly changed.